simulation:
  simple_planning_simulator:
    ros__parameters:
      loop_rate: 50.0
      simulation_frame_id: base_link
      map_frame_id: map
      initialize_source: RVIZ
      use_trajectory_for_z_position_source: true

      acc_time_constant: 0.1
      acc_time_delay: 0.1
      accel_rate: 7.0
      add_measurement_noise: true
      angvel_lim: 3.0
      angvel_noise_stddev: 0.0
      angvel_rate: 1.0
      angvel_time_constant: 0.5
      angvel_time_delay: 0.2
      initial_engage_state: true
      pos_noise_stddev: 0.01
      rpy_noise_stddev: 0.0001
      x_stddev: 0.0001 # x standard deviation for dummy covariance in map coordinate
      y_stddev: 0.0001 # y standard deviation for dummy covariance in map coordinate
      sim_steering_gear_ratio: 15.0
      steer_lim: 1.0
      steer_noise_stddev: 0.0001
      steer_rate_lim: 5.0
      steer_time_constant: 0.27
      steer_time_delay: 0.24
      tread_length: 1.0

      # Option for vehicle_model_type:
      # - IDEAL_STEER : reads velocity command. The steering and velocity changes exactly the same as commanded.
      # - DELAY_STEER : reads velocity command. The steering and velocity changes with delay model.
      # - DELAY_STEER_ACC : reads acceleration command. The steering and acceleration changes with delay model.
      vehicle_model_type: DELAY_STEER_ACC
      vel_lim: 50.0
      vel_noise_stddev: 0.0
      vel_time_constant: 0.61
      vel_time_delay: 0.25
      deadzone_delta_steer: 0.00
